canvas_modules/harness/test_resources/parameterDefs/oneofselect_paramDef.json

```diff
@@ -45,6 +45,7 @@
         "blue",
         "purple"
       ],
+      "enableIconCallBack": true,
```

Comment:
I don't think we'd add anything to the paramDef.  This would be a change in the test harness to implement the callback above.